src/imoinfo.h

```diff
@@ -492,6 +493,10 @@ class QDLLEXPORT IOrganizer: public QObject
    */
   virtual bool onPluginDisabled(const QString& pluginName, std::function<void()> const& func) = 0;
 
+protected:
+
+  using QObject::QObject;

```

Comment:
?

Reply:
Oops, that should not be here 🙃

Reply:
That was to set the parent of `OrganizerProxy` in `modorganizer` but I'm using `setParent` instead.